monarch_mapping_commons.Makefile

```diff
@@ -12,41 +12,51 @@ METADATA_DIR = 					metadata
 RUN = poetry run
 
 
+benchmark:
+	pip install py-spy
+	sudo py-spy record -o flamegraph.svg -- $(SSSOM_TOOLKIT) validate $(MAPPING_DIR)/gene_mappings.sssom.tsv
+
+
 $(MAPPING_DIR)/ $(SCRIPT_DIR)/ $(SRC_DIR)/ $(TMP_DIR)/:
 	mkdir -p $@
 
+
 $(MAPPING_DIR)/mondo.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
+
 $(MAPPING_DIR)/mesh_chebi_biomappings.sssom.tsv:
 	mkdir -p $(MAPPING_DIR) $(TMP_DIR)
 	wget https://raw.githubusercontent.com/biopragmatics/biomappings/master/docs/_data/sssom/biomappings.sssom.tsv -O $(TMP_DIR)/biomappings.sssom.tsv
 	$(RUN) python3 $(SCRIPT_DIR)/process_biomappings.py --input $(TMP_DIR)/biomappings.sssom.tsv --output $(TMP_DIR)/mesh_chebi_biomappings.sssom.tsv
 	$(RUN) sssom parse $(TMP_DIR)/mesh_chebi_biomappings.sssom.tsv -m $(METADATA_DIR)/mesh_chebi_biomappings.sssom.yml --prefix-map-mode merged -o $@
 
+
 $(MAPPING_DIR)/gene_mappings.sssom.tsv:
+ifndef GH_ACTION
```

Comment:
wrong conditional. if `GH_ACTION=false` this conditional evaluates to TRUE.

Reply:
I think you want this to be `ifneq ($(GH_ACTION), true)` (so this could is run _unless_ GH_ACTION variable is set to `true`.

Reply:
I tried that but it didn't work. Realistically, I don't expect anyone to be setting the GH_ACTION flag at all, so it should be fine. 
But I'll try again I suppose.